2023_VillaIslas_Science/2023_VillaIslas_Science.janno

```diff
@@ -0,0 +1,28 @@
+333C_TOL_a_MNT	F	Toluquilla;CentralMexico;SierraGorda	Mexico	MX	Querétaro	Toluquilla	20.88378	-99.53097	contextual	1301	1351	1401	Dates are taken from Date (CE) in S15	B2c	n/a	tooth	1	mtDNA-genome	Shotgun	minus	ds	haploid	https://github.com/nf-core/eager/releases/tag/2.4.6	94.499718	0	0.21912157243387528	n/a	n/a	n/a	Nr Snps (per library): 0. Estimate and error are weighted means of values per library. Libraries with fewer than 100 SNPs used in contamination estimation were excluded.	PRJEB51440;ERS10903723;ERR9576915	2023VillaIslasScience	333C_TOL_a	333C_TOL_a	0.0	0.0	0.0	0.0
+333O_TOL_b_MNT	U	Toluquilla;CentralMexico;SierraGorda	Mexico	MX	Querétaro	Toluquilla	20.88378	-99.53097	contextual	650	700	750	Dates are taken from Date (CE) in S5	B2l	n/a	tooth	1	mtDNA-genome	Shotgun	minus	ds	haploid	https://github.com/nf-core/eager/releases/tag/2.4.6	97.512945	0	0.31377645748236166	n/a	n/a	n/a	Nr Snps (per library): 0. Estimate and error are weighted means of values per library. Libraries with fewer than 100 SNPs used in contamination estimation were excluded.	PRJEB51440;ERS10903713;ERR9576916	2023VillaIslasScience	333O_TOL_b	333O_TOL_b	0.0	0.0	0.0	0.0
+333Q_TOL_b_MNT	U	Toluquilla;CentralMexico;SierraGorda	Mexico	MX	Querétaro	Toluquilla	20.88378	-99.53097	contextual	480	520	560	Dates are taken from Date (CE) in S10	A2d	n/a	tooth	1	mtDNA-genome	Shotgun	minus	ds	haploid	https://github.com/nf-core/eager/releases/tag/2.4.6	98.044843	0	0.20524344569288389	n/a	n/a	n/a	Nr Snps (per library): 0. Estimate and error are weighted means of values per library. Libraries with fewer than 100 SNPs used in contamination estimation were excluded.	PRJEB51440;ERS10903718;ERR9576917	2023VillaIslasScience	333Q_TOL_b	333Q_TOL_b	0.0	0.0	0.0	0.0
+37AI_R_b_MNT	U	Ranas;CentralMexico;SierraGorda	Mexico	MX	Querétaro	Ranas	20.92612	-99.56488	n/a	n/a	n/a	n/a	Dates are taken from Date (CE) in S9	A2d	n/a	tooth	1	mtDNA-genome	Shotgun	minus	ds	haploid	https://github.com/nf-core/eager/releases/tag/2.4.6	96.207416	0	0.3249466950959488	n/a	n/a	n/a	Nr Snps (per library): 0. Estimate and error are weighted means of values per library. Libraries with fewer than 100 SNPs used in contamination estimation were excluded.	PRJEB51440;ERS10903717;ERR9576918	2023VillaIslasScience	37AI_R_b	37AI_R_b	0.0	0.0	0.0	0.0
```

Comment:
`Date_Note` says "Dates are taken from Date (CE) in S9", but there is no date information. This also applies to some other samples here. What does "S9" mean, btw.?

Reply:
Thanks for pointing this oout. they should all read "S1". There is an excel autofill error in the original janno. S1 refers to "Supplementary Table S1". Changed the wording to make that clearer.

Reply:
In the cases where the Date info is `n/a`, I leave the Note in to denote that there is indeed no information on the Supp Table (i.e. it's left blank on purpose)